portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -14,25 +14,28 @@
 
 package com.google.sps.data;
 
+import com.google.appengine.api.blobstore.BlobKey;
+
 /**
  * Class representing a comment created on server-dev.html.
  *
  * <p>Note: The private variables in this class are converted into JSON.
  */
 public class Comment {
 
-  /** The strings of text for the comment */
+  /** The strings of text for the comment. */
```

Comment:
Optional: "The contents of the comment." might be clearer. This is 100% fine, though, as-is.

Reply:
I like it more as well. Changed.